src/firmware/main/world/wheel.c

```diff
@@ -0,0 +1,27 @@
+#include "world/wheel.h"
+
+#include <stdlib.h>
+
+struct Wheel
+{
+    void (*apply_wheel_force)(float force_in_newtons);
```

Comment:
Having wheel force as a 'Wheel' member seems strange to me, but I guess it makes the most sense for controlling the force output. Maybe throw in the Wheel position as well.

Reply:
The "wheel" abstracts any operation on that wheel. The reason it's force currently is that is what we use in firmware currently, and this layer is intended initially just to be an abstraction for our current firmware primitive structure, so that it can be ported to the simulator / new robots. I agree we will need position here eventually, but let's leave it out for now.